From c434437890ee3fd949830b294955318863edce3b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Tue, 30 Sep 2014 07:55:11 -0400
Subject: [PATCH] dmsquash-live: treat cancelled check as successfull

Also remove JobTimeout=0 setting, it is the default for Type=oneshot.

https://bugzilla.redhat.com/show_bug.cgi?id=1147941
---
 modules.d/90dmsquash-live/checkisomd5@.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/90dmsquash-live/checkisomd5@.service b/modules.d/90dmsquash-live/checkisomd5@.service
index 20e40bd7c8..1efb4a7b45 100644
--- a/modules.d/90dmsquash-live/checkisomd5@.service
+++ b/modules.d/90dmsquash-live/checkisomd5@.service
@@ -10,4 +10,4 @@ ExecStart=/bin/checkisomd5 --verbose %f
 StandardInput=tty-force
 StandardOutput=inherit
 StandardError=inherit
-TimeoutSec=0
+SuccessExitStatus=2
-- 
1.9.3

